COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2022 University of Washington
```

Comment:
It's not only allowed, it's required: see https://developer.lsst.io/legal/copyright-overview.html. (This is a good time to review the rules, since we're going to have to add a lot of 2023 copyrights to different packages!)

Reply:
Ah, but it looks like I misremembered the rules myself:

> If you move source files between different source repositories, remember to also audit and move copyright assignments as necessary.

Will reconstruct what was transferred from `ap_verify_ci_hits2015` and `ap_verify_ci_dc2` and fix.